libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2127,6 +2130,67 @@ void GCS_MAVLINK::send_raw_imu()
 #endif
 }
 
+#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
```

Comment:
```suggestion
#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED && AP_INERTIALSENSOR_ENABLED
```

Reply:
This is redundant because of a latter change